Source/ThirdParty/ANGLE/src/common/system_utils.cpp

```diff
@@ -226,7 +226,7 @@ void *OpenSystemLibraryAndGetError(const char *libraryName,
     {
         libraryWithExtension += dotExtension;
     }
-#if ANGLE_PLATFORM_IOS
+#if defined(ANGLE_PLATFORM_APPLE_EMBEDDED)
```

Comment:
Sometimes it's written `#if ANGLE_PLATFORM_APPLE_EMBEDDED` and other times `#if defined(ANGLE_PLATFORM_APPLE_EMBEDDED)`. We should pick one and be consistent.

Reply:
It should be safe to use `#if ANGLE_PLATFORM_APPLE_EMBEDDED`. Geoff Lang from the ANGLE team pointed out in an earlier review that undefined declarations evaluate to 0.

Reply:
I wasn't arguing from safety but consistency of spelling. It's only a minor nit.